src/lib/routing-tables.js

```diff
@@ -235,103 +232,52 @@ class RoutingTables {
   }
 
   /**
-   * Find the best intermediate ledger (`nextLedger`) to use after `sourceLedger` on
-   * the way to `finalLedger`.
-   * This connector must have `[sourceLedger, nextLedger]` as a pair.
-   *
-   * @param {IlpAddress} sourceAddress
-   * @param {IlpAddress} finalAddress
-   * @param {String} finalAmount
-   * @returns {Object}
+   * @param {IlpAddress} ledgerA
+   * @param {IlpAddress} ledgerB
+   * @returns {Route}
    */
-  findBestHopForDestinationAmount (sourceAddress, finalAddress, finalAmount) {
-    const nextHop = this._findBestHopForDestinationAmount(sourceAddress, finalAddress, +finalAmount)
-    if (!nextHop) return
-    // sourceLedger is the longest known prefix of sourceAddress (likewise for
-    // finalLedger/finalAddress).
-    const sourceLedger = nextHop.bestRoute.sourceLedger
-    const finalLedger = nextHop.bestRoute.destinationLedger
-    const nextLedger = nextHop.bestRoute.nextLedger
-    const routeFromAToB = this._getLocalPairRoute(sourceLedger, nextLedger)
-    const isFinal = nextLedger === finalLedger
-    // CAUTION: The amount at the `destinationLedger` (or `finalLedger`) is confusingly
-    // called `finalAmount` here, but will be **renamed** to `destinationAmount` by
-    // https://github.com/interledgerjs/ilp-core/blob/v14.0.1/src/lib/core.js#L199
-    // As you can see, even more confusingly, this function does actually return a
-    // `destinationAmount` field, but that is a bit of a misnomer here.
-    // `nextAmount` would be a better name for that field, since it is the amount at
-    // the `nextLedger` (see `const routeFromAToB` above).
-    // It will be ignored by
-    // https://github.com/interledgerjs/ilp-core/blob/v14.0.1/src/lib/core.js#L199
-    // but I left it in, since it is being checked by the unit tests - MJ.
-    return omitUndefined({
-      isFinal: isFinal,
-      isLocal: nextHop.bestRoute.isLocal,
-      sourceLedger: sourceLedger,
-      sourceAmount: nextHop.bestCost && nextHop.bestCost.toString(),
-      destinationLedger: nextLedger,
-      destinationAmount: nextHop.bestCost && routeFromAToB.amountAt(nextHop.bestCost).toString(),
-      destinationCreditAccount: isFinal ? null : nextHop.bestHop,
-      finalLedger: finalLedger,
-      finalAmount: finalAmount, // CAUTION: this will be used as destinationAmount
-      minMessageWindow: nextHop.bestRoute.minMessageWindow,
-      liquidityCurve: nextHop.bestRoute.curve && nextHop.bestRoute.curve.getPoints(),
-      additionalInfo: isFinal ? nextHop.bestRoute.additionalInfo : undefined
-    })
+  getLocalRoute (ledgerA, ledgerB) {
+    return this._getRoute(ledgerA, ledgerB, PAIR)
   }
 
   /**
-   * @param {IlpAddress} sourceAddress
-   * @param {IlpAddress} finalAddress
+   * @param {IlpAddress} source
+   * @param {IlpAddress} destination
```

Comment:
Why did this variable name change? If prefixes are now allowed then the `{IlpAddress}` type is no longer correct